cmd/config/internal/commands/cmdset.go

```diff
@@ -173,7 +173,7 @@ func (r *SetRunner) executeCmd(w io.Writer, pkgPath string) error {
 			fmt.Fprintf(w, "%s\n", err.Error())
 		}
 	} else {
-		fmt.Fprintf(w, "set %d field(s)\n", count)
+		fmt.Fprintf(w, "set %d field(s) of setter %q to value %q\n", count, r.Set.Name, r.Set.Value)
```

Comment:
Would this end up being a bit verbose with nested packages?

Reply:
I don't think so. This information is needed to give accurate status to the users.